public/network_vis_controller.js

```diff
@@ -497,7 +472,7 @@ module.controller('KbnNetworkVisController', function ($scope, $sce, $timeout, P
                         var usedColors = [];
 
                         // Check if "Node Color" is the last selection
-                        if (colorNodeAggId <= secondBucketAggId) {
+                        if (colorBucketId <= secondBucketId) {
```

Comment:
This error had the potential to be incorrectly thrown on Node-Relation with Color Node. It was comparing the agg ID, and in one case I saw the Relation have an agg ID of 7, and Node Color's was lower. Comparing the first column number seems safer, so this lexigraphical comparison works (ex: comparing relation `"col-2-6"` vs node color `"col-4-7"`.